website/content/docs/upgrading/upgrade-to-1.9.x.mdx

```diff
@@ -0,0 +1,90 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.9.x - Guides
+description: |-
+  This page contains the list of deprecations and important or breaking changes
+  for Vault 1.9.x. Please read it carefully.
+---
+
+# Overview
+
+This page contains the list of deprecations and important or breaking changes
+for Vault 1.9.x compared to 1.8. Please read it carefully.
+
+## Identity Tokens
```

Comment:
This file also has different content in `main`.

Reply:
Thanks for catching that, I might have messed up during merge conflict... I'll fix this up.

Reply:
Updated, should be good now!